dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -86,8 +90,18 @@ public static Path generate()
 
     ArtifactCache cache = loadArtifactInfo(managedDependencies);
     
+    ImmutableList<Path> classpath = StaticLinkageChecker.artifactsToClasspath(managedDependencies);
+    // TODO(suztomo): to take command-line option to choose entry point classes for reachability
+    ImmutableSet<Path> entryPoints = ImmutableSet.of(classpath.get(0));
+
+    boolean onlyReachable = false;
+    StaticLinkageChecker staticLinkageChecker =
+        StaticLinkageChecker.create(onlyReachable, classpath, entryPoints);
+    StaticLinkageCheckReport report = staticLinkageChecker.findLinkageErrors();
+
     List<ArtifactResults> table = generateReports(configuration, output, cache);
-    generateDashboard(configuration, output, table, cache.getGlobalDependencies());
+    generateDashboard(configuration, output, table, cache.getGlobalDependencies(),
+        report.toString());
```

Comment:
I think it would be more consistent to pass in a data object (`StaticLinkageCheckReport`) and then call the `toString()` inside `generateDashboard`.